samples/app-hello-world/nodejs/appManifest/manifest.json

```diff
@@ -1,80 +1,119 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.16",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
   "version": "1.0.0",
   "id": "${{TEAMS_APP_ID}}",
   "packageName": "com.contoso.helloworld",
   "developer": {
-    "name": "Contoso",
-    "websiteUrl": "https://www.microsoft.com",
-    "privacyUrl": "https://www.microsoft.com/privacy",
-    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+      "name": "Contoso",
+      "websiteUrl": "https://www.microsoft.com",
+      "privacyUrl": "https://www.microsoft.com/privacy",
+      "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
   },
   "name": {
-    "short": "Hello World",
-    "full": "Hello World App for Microsoft Teams"
+      "short": "ElementRelationshipSet",
```

Comment:
Is this an app for ElementRelationshipSet or for Hello World?

Reply:
Created New sample, Comment fixed